### PR TITLE
Remove dependency on bazel

### LINKED ISSUE #526: `buildifier` rule fetches the whole of https://github.com/bazelbuild/bazel.git
I sadly don’t have a CLI output because bazel makes it vanish after the fetch is complete (and you can’t copy auto-updating terminal).

Our rule looks like this:
```
# Run this to fix the errors in BUILD files.
buildifier(
    name = "buildifier-fix",
    mode = "fix",
    verbose = True,
)
```

and it starts fetches the bazel repo.

The whole repo takes >200s on our office connection and is basically impossible to wait on on a less potent one (>5 minutes on 1Mb/s). That’s bad.

My checkout of bazel is 750Mb, it’s a bit silly to download all that just to lint a bit of pseudo-python code.

### PR DESCRIPTION
**Pull request by vladmos**
The protofiles are copied from bazelbuild/bazel to avoid depending on a heavy repository.

Fixes #526

### CODE REVIEW

File: extra_actions_base_proto/BUILD.bazel [hunk 1]
```diff
@@ -7,14 +7,6 @@ go_proto_checkedin_test(
     src = "extra_actions_base.gen.pb.go",
 )
 
-genrule(
```
Comment (pmbethe09) [inline extra_actions_base_proto/BUILD.bazel]: Should we consider leaving a version of this that is not in the main codepath, but makes it easy to 're-vendor'?

File: extra_actions_base_proto/BUILD.bazel [hunk 2]
```diff
@@ -7,14 +7,6 @@ go_proto_checkedin_test(
...
-    name = "copy_and_fix",
-    srcs = ["@io_bazel//src/main/protobuf:srcs"],
-    outs = ["extra_actions_base.proto"],
-    cmd = ("SRCS=($(locations @io_bazel//src/main/protobuf:srcs));" +
-           "cp $$(dirname $$SRCS)/extra_actions_base.proto $@"),
-)
-
 proto_library(
     name = "blaze_proto",
     srcs = ["extra_actions_base.proto"],
```
Comment (pmbethe09) [inline extra_actions_base_proto/BUILD.bazel]: Now that it is vendored -- should call out via comment the exact bazel version/commit that this came from

File: deps_proto/BUILD.bazel [hunk 3]
```diff
@@ -7,14 +7,6 @@ go_proto_checkedin_test(
...
-    name = "copy_and_fix",
-    srcs = ["@io_bazel//src/main/protobuf:srcs"],
-    outs = ["deps.proto"],
-    cmd = ("SRCS=($(locations @io_bazel//src/main/protobuf:srcs));" +
-           "cp $$(dirname $$SRCS)/deps.proto $@"),
-)
-
 proto_library(
     name = "blaze_deps_proto",
     srcs = ["deps.proto"],
```
Comment (pmbethe09) [inline deps_proto/BUILD.bazel]: same comments as below -- add version/commit as a comment, should we leave a variation on copy_and_fix that writes to 'deps.proto.vendor' and then a shell script that can be run to clone, copy and replace the vendored files?

**Comment by ash2k** [comment]
This also fixes https://github.com/bazelbuild/buildtools/issues/143

**Comment by keith** [comment]
Any update on this?

File: extra_actions_base_proto/BUILD.bazel (hunk 1 above)
Comment (vladmos) [reply extra_actions_base_proto/BUILD.bazel]: Do you have an example of what you mean? How do I remove io_bazel from the external dependencies but still make it easy to re-vendor?

File: extra_actions_base_proto/BUILD.bazel (hunk 2 above)
Comment (vladmos) [reply extra_actions_base_proto/BUILD.bazel]: Added a comment.

**Pull request merged**